--- a/drivers/power/supply/mediatek/battery/mtk_battery.c
+++ b/drivers/power/supply/mediatek/battery/mtk_battery.c
@@ -205,19 +205,7 @@
 
 bool is_recovery_mode(void)
 {
-	int boot_mode = get_boot_mode();
-
-	if (is_fg_disabled())
-		return false;
-
-	bm_debug("mtk_battery boot mode =%d\n", boot_mode);
-	if (boot_mode == RECOVERY_BOOT) {
-		gm.log_level = BMLOG_DEBUG_LEVEL;
-		fg_cust_data.daemon_log_level = BMLOG_DEBUG_LEVEL;
-		return true;
-	}
-
-	return false;
+	return true;
 }
 
 bool is_fg_disabled(void)
